components/pricing/PricingTable.tsx

```diff
@@ -0,0 +1,303 @@
+import { Row, Col, Table } from "react-bootstrap";
+import Image from "../Image";
+export const PricingTable = () => {
```

Comment:
This could be abstracted further and pulled in from json or something, like how our comparison pages do it.  For now, because there were no major content changes, I only went as far as moving it to its own component to simplify the code for the page itself.

Reply:
Pulling this out 👍